# HG changeset patch
# User Brian <runningnak3d@gmail.com>
# Date 1277925263 21600
# Branch trunk
# Node ID f401c131f641361ed47d3ad736ab8d27363a405c
# Parent  d8bc896b17cf63c4275955609e4b6bb7678b10ee
* Fix authentication for the new client
* Thanks TOM_RUS

diff -r d8bc896b17cf -r f401c131f641 src/server/game/Server/WorldSocket.cpp
--- a/src/server/game/Server/WorldSocket.cpp	Wed Jun 30 13:11:44 2010 -0600
+++ b/src/server/game/Server/WorldSocket.cpp	Wed Jun 30 13:14:23 2010 -0600
@@ -768,7 +768,7 @@
     // NOTE: ATM the socket is singlethread, have this in mind ...
     uint8 digest[20];
     uint32 clientSeed;
-    uint32 unk2, unk3;
+    uint32 unk2, unk3, unk5, unk6, unk7;
     uint64 unk4;
     uint32 BuiltNumberClient;
     uint32 id, security;
@@ -797,6 +797,7 @@
     recvPacket >> account;
     recvPacket >> unk3;
     recvPacket >> clientSeed;
+    recvPacket >> unk5 >> unk6 >> unk7;
     recvPacket >> unk4;
     recvPacket.read (digest, 20);
 
